aten/src/THC/THCSortUtils.cuh

```diff
@@ -142,7 +142,7 @@ bitonicSortKVInPlace(TensorInfo<K, IndexType> keys,
                      IndexType keySliceStride,
                      TensorInfo<V, IndexType> values,
                      IndexType valueSliceStride,
-                     const Comparator& comp) {
+                     const Comparator comp) {
```

Comment:
This change makes me nervous.

1. There are a lot of other `const Comparator&` arguments in this file; why is only this one changed?

2. What if Comparator is not cheap to copy? Now you've hugely pessimized this function.